scripts/updateSources.js

```diff
@@ -159,10 +128,6 @@ async function main() {
       return content;
     }
 
-    // Dialog: Workaround for https://github.com/vaadin/vaadin-confirm-dialog-flow/issues/136
-    // Since this project contains a dependency to vaadin-confirm-dialog, the height is different
-    // and the tests fail.
-    content = ignore_test_method(content, source, 'DialogIT.openAndCloseBasicDialog_labelRendered');
```

Comment:
The underlying issue was fixed, see https://github.com/vaadin/vaadin-confirm-dialog-flow/issues/136#issuecomment-687126222